components/nexus-blobstore-file/pom.xml

```diff
@@ -48,6 +48,12 @@
       <artifactId>goodies-testsupport</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.sonatype.nexus</groupId>
+      <artifactId>nexus-test-concurrent</artifactId>
```

Comment:
`nexus-test-concurrent` currently depends on `nexus-common` but I didn't see anything nexus specific in the code, which makes me wonder if this could be moved to the goodies project so other projects could take advantage of it?